tests/monotouch-test/PassKit/PassTest.cs

```diff
@@ -38,7 +38,7 @@ public void Defaults ()
 				Assert.NotNull (pass.Icon, "Icon");
 #endif
 				Assert.Null (pass.LocalizedDescription, "LocalizedDescription");
-				Assert.That (pass.LocalizedName, Is.EqualTo ("Coupon"), "LocalizedName");
+				Assert.That (string.IsNullOrEmpty (pass.LocalizedName), Is.False, "LocalizedName");
```

Comment:
minor (no need to change it) but there's a simpler way
```
Assert.NotNull (pass.LocalizedName, "LocalizedName");
```

Reply:
Yes, but I was thinking, is this ever going to be "" ever? That might seem strange for the API user, and I did not want us doing something wrong, returning "" and end up with issues. (mainly, "I don't want me changing this ever to do something stupid" ;) )